### PR TITLE
dbdevops faqs page

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \_dbdevops faqs page 
* Jira/GitHub Issue numbers (if any): \DBOPS-778
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/faqs/database-devops-faqs.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+This article addresses some frequently asked questions about Harness Database DevOps (DB DevOps).
+
+### What is DB DevOps?
+
+Harness Database DevOps (DB DevOps) is a new module that integrates database changes into your deployment pipeline and provides greater visibility into database changes. It will allow you to manage database code like application code.
+
+### When will DB DevOps be available?
+
+The Alpha version became available in June 2024, enter Beta early august, and go GA in October 2024.
```
Comment (archit-harness) [inline docs/faqs/database-devops-faqs.md:15]: Is GA date right?
@stephenatwell ?

File: docs/faqs/database-devops-faqs.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+This should be there as a best practice and then executed to allow for any schema evolution. We are also exploring governance guard-rails to enforce such practices.
+
+### How does DB DevOps handle database provisioning?
+
+Initially we will not add functionality for this, but it is under consideration for our post-GA roadmap.
+
+### Does DB DevOps leverage any existing open-source tooling?
+
+Yes, we currently use liquidbase, and plan to add support for flyway in the future.
```
Comment (archit-harness) [inline docs/faqs/database-devops-faqs.md:49]: liquibase spelling is wrong

File: docs/faqs/database-devops-faqs.md (hunk 1 above)
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:15]: I would remove this..... has no business being in the documentation.

File: docs/faqs/database-devops-faqs.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Harness Database DevOps (DB DevOps) is a new module that integrates database changes into your deployment pipeline and provides greater visibility into database changes. It will allow you to manage database code like application code.
+
+### When will DB DevOps be available?
+
+The Alpha version became available in June 2024, enter Beta early august, and go GA in October 2024.
+
+### Will DB DevOps be a paid module?
+
+Yes.
```
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:19]: please remove.... all harness modules are paid modules.

File: docs/faqs/database-devops-faqs.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### How will DB DevOps be priced?
+
+The plan is to price around the concept of ‘database instances', which is the application of a particular database schema, to a particular database connection. This pricing metric aligns well to perceived customer value. For our startup tier we also plan to have per-user pricing, with a cap on the number of database instances per user.
```
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:23]: please remove. when we make this public we would update the harness-wide pricing documentation page with it.

File: docs/faqs/database-devops-faqs.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+### Does DB DevOps support rollbacks?
```
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:25]: please remove references to liquibase as much as possible. we're talking about DB DevOps, not liquibase (but yes, that is how it is all immplemented).

File: docs/faqs/database-devops-faqs.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+The reason customers may have asked how this works is if they felt the changes may potentially be destructive, like a column deletion. Once a column is dropped, its data is typically lost. Liquibase can recreate the column structure during rollback, but cannot automatically restore the lost data. 
+
+The way to mitigate this is to again use changesets to back up the data before deletion have rollback instructions to recreate the column and restore the data and cleans up the backup table.
+
+This should be there as a best practice and then executed to allow for any schema evolution. We are also exploring governance guard-rails to enforce such practices.
+
+### How does DB DevOps handle database provisioning?
+
+Initially we will not add functionality for this, but it is under consideration for our post-GA roadmap.
```
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:45]: I would remove this.

File: docs/faqs/database-devops-faqs.md (hunk 2 above)
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:49]: I would remove this.

File: docs/faqs/database-devops-faqs.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+### Does DB DevOps leverage any existing open-source tooling?
+
+Yes, we currently use liquidbase, and plan to add support for flyway in the future.
+
+### How does DB DevOps manage database changes in relation to application deployments?
+
+Both change types can be included in the same pipeline to ensure that the database and schema are deployed together.
```
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:53]: It's probably worth explicitly stating that customers can only deploy application changes if they are also licensed for Harness CD.

File: docs/faqs/database-devops-faqs.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+### How will Harness help my SQL performance?
+
+Harness will have views correlating changes in database performance metrics with changes that may affect it. Our roadmap also includes analyzing SQL queries to warn users of queries that may cause performance issues or outages before applying them to production.
```
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:57]: considering this functionality does not exist to date, I would remove this from the FAQ.

File: docs/faqs/database-devops-faqs.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+### Will DB DevOps be aware of changes done to my database outside of Harness?
+
+Initially DB DevOps will be aware of any changes done to your database by Liquibase. Eventually we plan to add detection for changes done that do not use Harness or Liquibase.
```
Comment (stephenatwell) [inline docs/faqs/database-devops-faqs.md:61]: please remove this question.

**Review by wei-harness — APPROVED** [review]
The md file looks good to me, need the module/folder owners to review the content.

**Pull request closed without merging**